README.md

```diff
@@ -29,11 +29,15 @@ compatibility!
 ### Screenshots
 ![Preview](retro-go-preview.jpg)
 
+# Recovery Mode
+### Reset Data and Boot to Launcher
+  1. Power up the device while holding down B.
+  2. Select ok and press A to conform it.
 
 # Installation
 ### ODROID-GO
   1. Download `retro-go_1.x_odroid-go.fw` from the release page and copy it to `/odroid/firmware` on your sdcard.
-  2. Power up the device while holding down B.
```

Comment:
On the odroid-go this is definitely button B on both the multi-firmware and the original firmware (It's controlled by CONFIG_BOOTLOADER_NUM_PIN_FACTORY_RESET over there).